docker-compose.yaml

```diff
@@ -590,6 +590,7 @@ services:
       # Typically when using the Docker Compose profile 'metrics' you want to turn this on
       - SUBSYSTEM_OPENTELEMETRY__SAMPLER_TYPE=${SUBSYSTEM_OPENTELEMETRY__SAMPLER_TYPE:-off}
       - SUBSYSTEM_OPENTELEMETRY__ENDPOINT=http://otel-collector:4317
+      - NOTIFICATIONS_DISABLED=true
```

Comment:
I generally prefer 'positive' / 'enabled' config properties as in: `NOTIFICATIONS_ENABLED`. Which would then be enabled by default but set to false here. I think those are easier to read and deal with.

Reply:
I did this to be consistent with OpenZaak, but I'll remove it anyways :-)